Docker.DotNet/Endpoints/ContainerOperations.cs

```diff
@@ -385,27 +387,30 @@ public Task StartContainerExecAsync(string id, CancellationToken cancellationTok
             return this.Client.MakeRequestAsync(new[] { NoSuchContainerHandler }, HttpMethod.Post, path, null, data, null, cancellationToken);
         }
 
+        // StartAndAttachContainerExecAsync will start the process specified by id with stdin, stdout, stderr
+        // connected, and optionally using terminal emulation if tty is true.
         public async Task<MultiplexedStream> StartAndAttachContainerExecAsync(string id, bool tty, CancellationToken cancellationToken)
         {
+            return await StartWithConfigContainerExecAsync(id, new ExecConfig() { AttachStdin = true, AttachStderr = true, AttachStdout = true, Tty = tty }, cancellationToken);
```

Comment:
I went ahead and changed the approach here... Since the method is called `StartAndAttachContainerExecAsync` I updated it to actually connect to all the pipes.  Otherwise, if the exec process was created with `AttachStdin` set to true and then it was passed as false here, we end up with that nasty hang.  It didn't seem write that calling this method might hang if the exec was configured a certain way, so I expect that this will get the right behavior.